### PR TITLE
New uploader readme

### PR DESCRIPTION
**Pull request by KamillaGarifullina**

### CODE REVIEW

**Review by paninebaluy — CHANGES_REQUESTED** [review]
* We don't use JS libraries as jQuery anymore --> We don't use JS libraries such as jQuery anymore
* Close to native web platform as possible --> As close to the native web platform as possible
* We are use Shadow DOM --> We use Shadow DOM
* etc --> etc.
* English will be a fallback --> English will be the fallback

File: uploader/README.md [hunk 1]
```diff
@@ -1,6 +1,123 @@
...
+* [Simplified case](./simplified/)
+
+All content fully accessible at https://uploadcare.github.io/jsdk/.
+
+## Features
+* JS size. The total amount of JavaScript code in web applications has become smaller.
+* Minimum of external dependencies. We don't use JS libraries such as jQuery anymore.
+* Modern web standards compatibility. We use Shadow DOM or CSP in the integration environment.
+* Build stage is not necessary for development. Use the code directly in any modern browser or Node.js.
+* As close to native web platform as possible. We don't force you to use any specific development environment tools.
```
Comment (rsedykh) [inline uploader/README.md:15]: Features of uploader? Or if this particular solution? Is SIZE is really #1 reason devs choose ready-made uploaders? What about customer facing features?

File: uploader/README.md [hunk 2]
```diff
@@ -1,6 +1,123 @@
...
+
+## Supported browsers
+Uploader is supported and tested in all major modern desktop and mobile browsers:
+
+* Chrome
+* Firefox
+* Safari
+* Edge
+* Opera
+* etc.
```
Comment (rsedykh) [inline uploader/README.md:25]: Versions?

File: uploader/README.md [hunk 3]
```diff
@@ -1,6 +1,123 @@
...
+
+[Internet Explorer](https://uploadcare.com/blog/uploadcare-stops-internet-explorer-support/) is outdated and not supported anymore.
+
+## Quick start
+### Public Key
+Obtain a Public API Key in your Uploadcare project's dashboard to use file uploading features.
```
Comment (rsedykh) [inline uploader/README.md:31]: URLs?

File: uploader/README.md [hunk 4]
```diff
@@ -1,6 +1,123 @@
...
+* Edge
+* Opera
+* etc.
+
+[Internet Explorer](https://uploadcare.com/blog/uploadcare-stops-internet-explorer-support/) is outdated and not supported anymore.
+
+## Quick start
+### Public Key
+Obtain a Public API Key in your Uploadcare project's dashboard to use file uploading features. 
+For demo-only purposes you can use demopublickey instead.
```
Comment (rsedykh) [inline uploader/README.md:32]: `demopublickey`?

File: uploader/README.md [hunk 5]
```diff
@@ -1,6 +1,123 @@
...
+
+### Integrate
+To integrate Uploadcare widget, first of all, you should connect our script:
```
Comment (rsedykh) [inline uploader/README.md:35]: Uploader widget? Not uc-uploader? Not upload blocks?

File: uploader/README.md [hunk 6]
```diff
@@ -1,6 +1,123 @@
...
+* Dropbox
+* Onedrive
+* Huddle 
+* Box
+* Instagram
+* Vk
+* Facebook
+* Flickr
+* Gphotos
+* Evernote
```
Comment (rsedykh) [inline uploader/README.md:113]: This looks like random order plus it doesn't look nice..

**Review by rsedykh — CHANGES_REQUESTED** [review]
I see it's not yet close, but still left some disturbing comments..

File: uploader/README.md (hunk 5 above)
Comment (KamillaGarifullina) [reply uploader/README.md]: Rewritten

File: uploader/README.md (hunk 6 above)
Comment (KamillaGarifullina) [reply uploader/README.md]: Rewritten

File: uploader/README.md (hunk 4 above)
Comment (KamillaGarifullina) [reply uploader/README.md]: Updated

File: uploader/README.md (hunk 2 above)
Comment (KamillaGarifullina) [reply uploader/README.md]: Rewritten

File: uploader/README.md (hunk 3 above)
Comment (KamillaGarifullina) [reply uploader/README.md]: Added

File: uploader/README.md [hunk 7]
```diff
@@ -1,6 +1,131 @@
...
+| **Desktop** | **Mobile**           |
++=============+======================+
+| Chrome: 37+ | Android Browser: 4.4+|
++-------------+----------------------+
+| Firefox: 32+| Opera Mobile: 8+     |
++-------------+----------------------+
+| Safari: 9+  | iOS Safari: 9+       |
++-------------+----------------------+
+| Edge: 12+   | IE Mobile: 11+       |
++-------------+----------------------+
```
Comment (rsedykh) [inline uploader/README.md:30]: Не билдится в табличку.

**Review by rsedykh — CHANGES_REQUESTED** [review]
Оставил пару комментариев к тому, к чему пока можно. Читается тяжело.

И, вопрос — из https://github.com/uploadcare/docs/pull/810/files все полезное взяли?

**Pull request merged**